tests/unit/experiments/experiment_tests.py

```diff
@@ -529,6 +529,88 @@ def test_that_bucketing_events_not_sent_if_cant_find_experiment(self):
         experiments.variant("test", user=self.user)
         self.assertEqual(self.event_logger.log.call_count, 0)
 
+    def test_none_returned_on_variant_call_with_bad_id(self):
+        self.mock_filewatcher.get_data.return_value = {
+            "test": {
+                "id": "1",
+                "name": "test",
+                "owner": "test_owner",
+                "type": "r2",
+                "version": "1",
+                "start_ts": time.time() - THIRTY_DAYS,
+                "stop_ts": time.time() + THIRTY_DAYS,
+                "experiment": {
+                    "id": 1,
+                    "name": "test",
+                    "variants": {
+                        "active": 50,
+                        "control_1": 25,
+                        "control_2": 25,
+                    }
+                }
+            }
+        }
+        experiments = Experiments(
+            config_watcher=self.mock_filewatcher,
+            server_span=self.mock_span,
+            context_name="test",
+            event_logger=self.event_logger,
+        )
+        self.assertEqual(self.event_logger.log.call_count, 0)
+        variant = experiments.variant("test", user=self.user)
+        self.assertEqual(variant, None)
+
+    def test_none_returned_on_variant_call_with_no_times(self):
+        self.mock_filewatcher.get_data.return_value = {
+            "test": {
+                "id": 1,
+                "name": "test",
+                "owner": "test_owner",
+                "type": "r2",
+                "version": "1",
+                "experiment": {
+                    "id": 1,
+                    "name": "test",
+                    "variants": {
+                        "active": 50,
+                        "control_1": 25,
+                        "control_2": 25,
+                    }
+                }
+            }
+        }
+        experiments = Experiments(
+            config_watcher=self.mock_filewatcher,
+            server_span=self.mock_span,
+            context_name="test",
+            event_logger=self.event_logger,
+        )
+        self.assertEqual(self.event_logger.log.call_count, 0)
+        variant = experiments.variant("test", user=self.user)
+        self.assertEqual(variant, None)
+
+    def test_none_returned_on_variant_call_with_no_experiment(self):
+        self.mock_filewatcher.get_data.return_value = {
```

Comment:
This is intended to test that a call to an unconfigured test doesn't raise, right?

Reply:
yep. If it raised, this would die. Since we're just catching all exceptions now, these tests are a bit redundant, but doesn't hurt to have extra tests.